contracts/multisig/src/signing.rs

```diff
@@ -25,19 +24,19 @@ impl SigningSession {
             id: session_id,
             key_id,
             msg,
+            state: MultisigState::Pending,
         }
     }
 }
 
 fn validate_session_signature(
     session: &SigningSession,
-    signatures: &HashMap<String, Signature>,
     key: &Key,
     signer: String,
```

Comment:
```suggestion
    signer: &str,
```
A validation method should not take away the ownership.

Reply:
changed this to `&Addr`